### PR TITLE
Added missing tests for locationresolver

### PR DESCRIPTION
**Pull request by VaibhavMalik4187**
Wrote new tests to cover the unhandled statements in the `NewFixPathLocationResolver` function.

### CODE REVIEW

**Comment by VaibhavMalik4187** [comment]
@matthyx Do you mind if I ask why is the code coverage action not working for the last couple of days? Did the API token expire?

**Comment by rotemamsa** [comment]
@yonatanamz is this relates to the token issue?

**Comment by VaibhavMalik4187** [comment]
Putting the code coverage action aside, I guess this PR is ready for a code review.

File: examples/online-boutique/invalid.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
```
Comment (matthyx) [inline examples/online-boutique/invalid.yaml:1]: can you change that file content with just `@` to clearly show it's invalid?

Comment (VaibhavMalik4187) [reply examples/online-boutique/invalid.yaml]: Done

**Review by matthyx — APPROVED** [review]

**Pull request merged**